brickstrap.sh

```diff
@@ -214,9 +238,10 @@ function brp_sanity_check_cli_options()
 
 function brp_validate_cli_options()
 {
-    brp_sanity_check_cli_options "$@" && \
-    brp_validate_project_name && \
+    brp_sanity_check_cli_options "$@"
+    brp_validate_project_name
     brp_validate_component_names
+    brp_validate_qemu || [ $? -eq 255 ] # no QEMU specified = 255
```

Comment:
I think what I would like to have happen here is that if `$QEMU_STATIC` or `-Q something` is specified, resolve it and make sure that a source qemu file actually exists. If qemu is not specified in any way, make sure `update-binfmts` exists by running `update-binfmt --version`. This way we can fail as early as possible in most cases.